### PR TITLE
have geth data files on the same place as lighthouse data

### PR DESCRIPTION
**Pull request by kvbik**
It would be nice to have `geth` data files also next to `docker-compose` configuration the same way we have there `lighthouse` data.

It is nice when in need of wiping or inspecting the data, so we don't need to dig deep somewhere in `/var/lib/docker/volumes/lighthousedocker_geth/_data/`.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ services:
         build:
             context: geth
         volumes:
-            - geth:/root/.ethereum
+            - ./geth-data:/root/.ethereum
```
Comment (AgeManning) [inline docker-compose.yml:30]: I think we should probably make it obvious that its here and not have it as a hidden directory

**Review by AgeManning — COMMENTED** [review]
Minor suggestion

File: docker-compose.yml (hunk 1 above)
Comment (hukkinj1) [reply docker-compose.yml:30]: @AgeManning perhaps you misread the path, and placement of the dot? `./geth-data` should not be a hidden directory

**Comment by hukkinj1** [comment]
With this change, can we remove
```
volumes:
    geth:
```
from the end of the file?

File: docker-compose.yml (hunk 1 above)
Comment (kvbik) [reply docker-compose.yml:30]: Like @hukkinj1 is saying. We are "mounting" local dir `geth-data` to the `/root/.ethereum` in the container. It is the default place of Geth datafiles. The same way `/root/.lighthouse` are. I am not a big fan of the hidden dirs there either, but that is how it is :D

**Comment by kvbik** [comment]
Oh, I've completely missed the `volumes:geth:` at the end of the file :D I believe it can be removed. Happy to do that if someone else confirms! `.gitignore` I am about to push the change!

**Comment by kvbik** [comment]
Ah, I've figured the old role of `volumes:geth:`. Really not needed any more. Removed. And merged latest master, so we are up to date with this PR.

**Review by AgeManning — APPROVED** [review]
Ah Yep. 

Sorry guys, misread the path :), last review must have been late at night.

**Pull request merged**